--- origsrc/promoe-0.1.0/src/src.pro	2009-08-24 17:43:09.000000000 -0500
+++ src/promoe-0.1.0/src/src.pro	2010-02-09 00:32:02.542849800 -0600
@@ -7,7 +7,7 @@ load (../features/pkgconfig.prf)
 !xmms2-client-cpp:error (Please install the xmms2 c++ client library)
 
 COMPONENTS+=../lib/liblib.a
-LIBS += $$COMPONENTS
+LIBS = $$COMPONENTS $$LIBS -lboost_signals-mt
 PRE_TARGETDEPS = $$COMPONENTS
 
 MOC_DIR     = .moc
@@ -17,7 +17,7 @@ UI_DIR      = .ui
 include(src.pri)
 
 TARGET = promoe
-DESTDIR = $$PWD/..
+DESTDIR = ..
 
 ;RESOURCES += ../data/promoe.qrc
 RESOURCES += ../data/Almond-blue.qrc
